stable_baselines3/common/logger.py

```diff
@@ -296,6 +333,12 @@ def write(self, key_values: Dict[str, Any], key_excluded: Dict[str, Union[str, T
             if isinstance(value, Video):
                 self.writer.add_video(key, value.frames, step, value.fps)
 
+            if isinstance(value, Figure):
+                self.writer.add_figure(key, value.figure, step, close=value.close)
+
+            if isinstance(value, Image):
```

Comment:
While you are at it it would be great to also support [adding text](https://pytorch.org/docs/stable/tensorboard.html#torch.utils.tensorboard.writer.SummaryWriter.add_text) here?

Reply:
what would be the use of it?
for debugging?

Reply:
Yes, I think it will be mainly useful for debugging and visualization. I am training a biomechanical model that types on a simulated keyboard and would like to see what it types. It may also be useful when developing RL based language models, autocorrection etc.

Reply:
I would be ok with it but in an other PR (as we would need to add support for the other logger output too)
Please open a feature request issue first ;) (so we can also discuss alternatives)

Reply:
I opened #286

Reply:
It would also be useful to log the graph (representation of the agent) at the beginning of the learn method.